Description: Make test require rspec-its
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2015-09-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -7,6 +7,7 @@
 require 'rspec/its'
 require 'redis'
 require 'logger'
+require 'rspec/its'
 
 $TESTING=true
 $:.unshift File.join(File.dirname(__FILE__), '..', 'lib')
